### ISSUE TITLE
Open every file without storing it permanently

### ISSUE THREAD

**Issue by tobiasKaminsky** [nextcloud/android#838]
When clicking on a file it should be choosable if the file gets stored or just cached internally and displayed.
(feature request via email)

**Comment by mario** [comment]
No. Please don't. I'd always opt for just displaying it, and not any kind of questions/choices.
